demo/src/ModelInput.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import Button from './components/Button'
+import ModelIntro from './components/ModelIntro'
+
+
+// TODO: if you want to have some quickly-accessible examples, you can add them here.  You can then
```

Comment:
I think this is not quite the right guidance, because removing the dropdown altogether is a non-trivial change. I would either say something like "add a few examples here", or be more explicit about what to remove if you don't want a dropdown

Reply:
Thanks, good point.